### ISSUE TITLE
Install Error: Cancel was selected, exiting installer

### ISSUE THREAD

**Issue by frmoronari** [pi-hole/pi-hole#3750]
### Versions
- Pi-hole:  not installed
- AdminLTE:  not installed
- FTL:  not installed

### Platform
- OS and version:  Fedora 32
- Platform:  Raspeberry Pi 3B+

### Expected behavior

Perform the installation after downloading the script with wget -O basic-install.sh https://install.pi-hole.net and run sudo bash basic-install.sh

### Actual behavior / bug

When `sudo bash basic-install.sh` is executed Cancel was selected, exiting installer with no log.

When running with `sudo sh basic-install.sh`, during the OS check function, an unexpected token error is reported on line 191:
bash basic-install.sh: line 191: syntax error next to unexpected token `<'

### Steps to reproduce

1. Go to 'wget -O basic-install.sh https://install.pi-hole.net'
2. Execute 'sudo bash basic-install.sh' (or sudo sh bash basic-install.sh for see other error)
3. See error

### Screenshots

https://i.imgur.com/koLUGi5.png

### Additional context

The same installation process was successfully performed on Fedora 32 on Raspberry Pi 4B.

**Comment by yubiuser** [comment]
It hangs at the `os_check`

https://github.com/pi-hole/pi-hole/blob/6b536b7428a1f57ff34ddc444ded6d3a62b00a38/automated%20install/basic-install.sh#L191

Try

`curl -sSL https://install.pi-hole.net | PIHOLE_SKIP_OS_CHECK=true sudo -E bash`
as a workaround.

**Comment by frmoronari** [comment]
> It hangs at the `os_check`
> 
> https://github.com/pi-hole/pi-hole/blob/6b536b7428a1f57ff34ddc444ded6d3a62b00a38/automated%20install/basic-install.sh#L191
> 
> Try
> 
> `curl -sSL https://install.pi-hole.net | PIHOLE_SKIP_OS_CHECK=true sudo -E bash`
> as a workaround.

Yes, i read the script on de 191 line, but i don't have skill to comprehensive completely.
I trying the solution and back with response. Thanks.

**Comment by yubiuser** [comment]
@frmoronari 

Does the issue still persist?

**Comment by yubiuser** [comment]
Closing due to inactivity. Feel free to open a new report if you encounter the issue again.
